### PR TITLE
pihole: add page

### PR DESCRIPTION
**Pull request by rbuchberger**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Sorry for the extra commit-- initially I found a few articles about setting up a pihole on a mac, so I thought it belonged in common. The [officially supported operating systems](https://discourse.pi-hole.net/t/hardware-software-requirements/273) are all linux distros though, so I moved it there. 

I wasn't entirely sure how to best describe `status` vs `chronometer`. The former gives something like this (and immediately returns):

```
$ pihole status 
[sudo] password for robert: 
  [✓] DNS service is running
  [✓] Pi-hole blocking is Enabled
```

While the latter takes over the terminal and gives something like this, updated once a second:

```
|¯¯¯(¯)_|¯|_  ___|¯|___        Core: version is 4.2.1
| ¯_/¯|_| ' \/ _ \ / -_)        Web: v4.3.1
|_| |_| |_||_\___/_\___|        FTL: 
 ——————————————————————————————————————————————————————————
  Hostname: cloudBucket        (Arch  )
    Uptime: 12 days, 06:09:28
 Task Load: 0.11 0.10 0.07     (Active: 0 of 40 tasks)
 CPU usage: 0%                 (4x 2.7 GHz @ 316k)
 RAM usage: 30%                (Used: 1 GB of 4 GB)
 HDD usage: 24%                (Used: 7 GB of 31 GB)
  LAN addr: 192.168.1.2        (Gateway: 192.168.1.1)
DHCP usage: 4%                 (Leased: 7 of 156)
   Pi-hole: Active             (Blocking: 113445 sites)
 Ads Today: 2%                 (Total: 235 of 15560)
Local Qrys: 30%                (2 DNS servers)
   Blocked: youtube.com 
Top Advert: onesignal.com 
Top Domain:   
Top Client: charybdis
```

Feedback appreciated!

### CODE REVIEW

File: pages/linux/pihole.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# pihole
+
+> Terminal interface for the Pi-Hole ad-blocking DNS server.
+> <https://pi-hole.net>.
```
Comment (pxgamer) [inline pages/linux/pihole.md]:
```suggestion
> More information: <https://pi-hole.net>.
```

File: pages/linux/pihole.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Check status:
```
Comment (pxgamer) [inline pages/linux/pihole.md]: What does this check the status of?

Comment (rbuchberger) [reply pages/linux/pihole.md]: The Pi-hole daemon. Pi-hole is an ad-blocking DNS server (meant to run on a raspi, can run on any linux machine), this command tells you if it's up and running or not. Its output looks like this:

```
╭─ robert@cloudBucket  ~  
╰─ pihole status
  [✓] DNS service is running
  [✓] Pi-hole blocking is Enabled
```

Comment (pxgamer) [reply pages/linux/pihole.md]: Maybe it would be better as:
```suggestion
- Check the status of the daemon:
```

Comment (rbuchberger) [reply pages/linux/pihole.md]: I tweaked it. New commit inbound!

File: pages/linux/pihole.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`pihole {{enable|disable}}`
+
+- Restart the daemon (not the server itself):
+
+`pihole restartdns`
+
+- Whitelist or blacklist a domain:
+
+`pihole {{whitelist|blacklist}} {{reddit.com}}`
```
Comment (agnivade) [inline pages/linux/pihole.md]: Let's only use example.com as sample domains.

**Comment by rbuchberger** [comment]
Apparently! 

https://pi-hole.net/2017/05/01/pi-hole-3-0-is-here-featuring-our-new-ftl-engine/

**Pull request merged**